task4/src/utils/input/input.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './input.scss';
+import PropTypes from 'prop-types';
+
+class Input extends React.Component {
```

Comment:
Why use class components instead of hooks? Do you already know hooks?

Reply:
I haven't covered  the hooks course when this was made. Will refactor this on the next PR